gradle/libs.versions.toml

```diff
@@ -0,0 +1,96 @@
+[versions]
+apache-commonsCompress = "1.23.0"
+apache-commonsIO = "2.13.0"
+apache-commonsLang = "3.12.0"
+awaitility = "4.2.0"
+bouncycastle = "1.75"
+byteBuddy = "1.14.5"
+concurrentUnit = "0.4.6"
+dropwizard-metrics = "4.2.19"
+eclipse-collections = "11.1.0"
+equalsVerifier = "3.14.2"
+findsecbugs = "1.12.0"
+guava = "32.0.1-jre"
+guice = "5.1.0"
+hivemq-extensionSdk = "4.21.0"
```

Comment:
we need to update the CE release card template to update the hivemq-extension-sdk version in this file now.